beetmoverscript/src/beetmoverscript/constants.py

```diff
@@ -69,7 +69,7 @@
 
 HASH_BLOCK_SIZE = 1024 * 1024
 
-RELEASE_BRANCHES = ("mozilla-central", "mozilla-beta", "mozilla-release", "mozilla-esr52" "comm-central", "comm-beta", "comm-esr60")
+RELEASE_BRANCHES = ("mozilla-central", "mozilla-beta", "mozilla-release", "mozilla-esr52comm-central", "comm-beta", "comm-esr60")
```

Comment:
Copy-pasting the comment here :)
I think this is not intentional.
Shouldn't it be `"mozilla-esr52", "comm-central"` ?

Reply:
argh, reformatting wasn't supposed to happen here...that's happening in https://github.com/mozilla-releng/scriptworker-scripts/pull/1116